VERSION

```diff
@@ -1,5 +1,5 @@
 VERSION="2.11.0"
```

Comment:
VERSION should not be modified (only if we do the release, not sure it goes in this PR)